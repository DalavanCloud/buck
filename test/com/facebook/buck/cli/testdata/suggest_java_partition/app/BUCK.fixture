java_library(
  name = 'app',
  srcs = glob(['*.java']),
  autodeps = True,
)
