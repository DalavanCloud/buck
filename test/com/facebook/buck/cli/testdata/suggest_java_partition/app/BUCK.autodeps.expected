#@# GENERATED FILE: DO NOT MODIFY 451cf7518f398403f999062a8f2edabf4a3c682b #@#
{
  "app" : {
    "deps" : [
      "//lib:lib.Bravo"
    ],
    "exported_deps" : [ ]
  }
}
